### PR TITLE
changes to protected create

### PR DESCRIPTION
**Pull request by pkamano**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -45,6 +45,7 @@ class App extends Component {
   }
 
   createUserDestinations = (mytrips) => {
+    console.log("mytrips", mytrips)
```
Comment (elysemontano) [inline app/javascript/components/App.js]: Let's remove this console log

File: app/javascript/components/pages/UserCreate.js [hunk 2]
```diff
@@ -8,33 +8,39 @@ class UserCreate extends Component {
...
+    alert('A destination was made!: ' + this.state.value);
+    this.props.createUserDestinations(this.state.newDestination)
+    this.setState({ submitted: true})
   }
 
 
 
   render() {
-
+  console.log("state", this.state)
```
Comment (elysemontano) [inline app/javascript/components/pages/UserCreate.js]: Let's remove this as well

File: app/javascript/components/pages/UserCreate.js [hunk 3]
```diff
@@ -84,30 +90,30 @@ class UserCreate extends Component {
               value = {this.state.newDestination.trip_details}
               />
         </FormGroup>
-        <FormGroup>
+        {/* <FormGroup>
```
Comment (elysemontano) [inline app/javascript/components/pages/UserCreate.js]: Please remove commented out code

**Review by elysemontano — CHANGES_REQUESTED** [review]
Left some comments

**Review by triciasykes — APPROVED** [review]

**Pull request merged**